utils/kata-manager.sh

```diff
@@ -838,10 +847,12 @@ handle_installation()
 	# These params can be blank
 	local kata_version="${7:-}"
 	local containerd_flavour="${8:-}"
+	local kata_tarball="${10:-}"
 
 	local install_docker="${9:-}"
 	[ -z "$install_docker" ] && die "no install docker value"
 
+
```

Comment:
Can you move the new param variable down below the `$9` one:

```suggestion
	local install_docker="${9:-}"
	[ -z "$install_docker" ] && die "no install docker value"

	local kata_tarball="${10:-}"
```